vortex-scan/src/row_mask.rs

```diff
@@ -50,14 +49,14 @@ impl RowMask {
     pub fn new_valid_between(begin: u64, end: u64) -> Self {
         let length =
             usize::try_from(end - begin).vortex_expect("Range length does not fit into a usize");
-        RowMask::new(Mask::from(BooleanBuffer::new_set(length)), begin)
+        RowMask::new(Mask::new_true(length), begin)
```

Comment:
this must have slipped in all the refactors, we immediately throw away that booleanbuffer